src/CSSBattle/12_WigglyMoustache/ZhangZhikai.html

```diff
@@ -0,0 +1,88 @@
+
+<div class="bgsemi">
+  <div class="innercircle middle"></div>
+  <div class="end"></div>
+</div>
+<div class="bgsemi rightsemi">
+  <div class="innercircle right"></div>
+  <div class="rightend"></div>
+</div>
+<div class="bgsemi leftsemi">
+  <div class="innercircle left"></div>
+  <div class="leftend"></div>
+</div>
+
+<style>
+  body {
+    background: #F5D6B4;
+  }
+  div {
+  }
+  .smallsemi {
+    width: 20px;
+    height: 20px;
+    border-radius: 100%;
+    position: absolute;
+    left: -75px;
+    top: 50px;
+  }
+  .bgsemi {
+	top: 100px;
+    border-radius: 50px 50px 0 0;
+    position: absolute;
+    left: 150px;
+    width: 100;
+    height: 50;
+    background: #D86F45;
+    z-index: 5;
+  }
+  .leftsemi {
+    transform: rotate(180deg);
+    left: 70px;
+    top: 150px;
+  }
+  .rightsemi {
+	transform: rotate(180deg);
+    left: 230px;
+    top: 150px;
+  }
+  .innercircle {
+    background: #F5D6B4;
+    width: 60px;
+    height: 30px;
+    border-radius: 30px 30px 0 0;
+    position: fixed;
+
+  }
+  .leftend {
+    width: 20px;
+    height: 10px;
+    border-radius: 0 0 10px 10px;
+    position: fixed;
+    top: 50px;
+    left: 80px;
+    background: #D86F45;
+  }
+  .rightend {
+    width: 20px;
+    height: 10px;
+    border-radius: 0 0 10px 10px;
+    position: fixed;
+    top: 50px;
+	background: #D86F45;
```

Comment:
Consider removing unnecessary indentation
```suggestion
  background: #D86F45;
```

Reply:
50ed92b92dcb6d7fac9a9b6965067d8e1abde559

Reply:
e635fab57fb3716abf4f1629b8369570f0a3a9e1